api/hook1_5.sample

```diff
@@ -1,7 +1,47 @@
 {
-  "id": 2361,
-  "hook": "request.add",
-  "subject": "Request subject",
-  "eventUser": 356,
-  "eventProject": 213
+    "hook": "dayreport.add",
+    "eventUser": -1,
+    "id": 1409,
+    "eventProject": -1,
+    "type": 1,
+    "url": "https://endurable-hound.glitch.me/hook"
+}
+
+Triggered hook would look as below 
+
+{
+	"hook": "project.update",
```

Comment:
should this be "dayreport.add" as above and the resource should be a dayreport OR change the above hook to "project.update"?

Reply:
done